arch/xtensa/src/esp32s2/Kconfig

```diff
@@ -718,6 +724,11 @@ config ESP32S2_SPIRAM_SPEED_80M
 
 endchoice # ESP32S2_SPIRAM_SPEED
 
+config ESP32S2_SPIRAM_SPEED
+	int
+	default 80 if ESP32S2_SPIRAM_SPEED_80M
+	default 40 if ESP32S2_SPIRAM_SPEED_40M
+
```

Comment:
This configuration seems to go unused in the PR, consider removing it.

Reply:
Fixed, now it is used to report the frequency.